plenum/test/primary_selection/test_add_node_to_pool_with_large_ppseqno.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from plenum.common.util import randomString
+from plenum.test.helper import sendReqsToNodesAndVerifySuffReplies
+from plenum.test.view_change.helper import ensure_several_view_change
+from plenum.test.pool_transactions.helper import addNewStewardAndNode
+from plenum.test.test_node import checkNodesConnected
+from plenum.test.node_catchup.helper import ensureClientConnectedToNodesAndPoolLedgerSame, waitNodeDataEquality
+
+
+def _get_ppseqno(nodes):
+    res = set()
+    for node in nodes:
+        for repl in node.replicas:
+            res.add(repl.lastPrePrepareSeqNo)
+    assert(len(res) == 1)
+    return min(res)
+
+
+def _set_ppseqno(nodes, new_ppsn):
+    for node in nodes:
+        for repl in node.replicas:
+            repl.lastPrePrepareSeqNo = new_ppsn
+            repl.h = new_ppsn
+            repl.last_ordered_3pc = (repl.viewNo, new_ppsn)
+
+
+@pytest.mark.parametrize('do_view_change', [0, 3, 5])
+def test_add_node_to_pool_with_large_ppseqno_diff_views(do_view_change, looper, txnPoolNodeSet, tconf, steward1,
+                                                        stewardWallet, tdir, client_tdir, allPluginsPath):
+
+    ensure_several_view_change(looper, txnPoolNodeSet, do_view_change)
+    sendReqsToNodesAndVerifySuffReplies(looper, stewardWallet, steward1, numReqs=3)
```

Comment:
Please use indy sdk for tests

Reply:
the test will use old client api for now untill node txn will be moved to sdk